src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -2495,7 +2493,7 @@ private static void ComputeDeclarationsInNode(SemanticModel semanticModel, ISymb
             var semanticModel = symbolEvent.SemanticModelWithCachedBoundNodes ??
                 GetOrCreateSemanticModel(decl.SyntaxTree, symbolEvent.Compilation);
 
-            var declarationAnalysisData = ComputeDeclarationAnalysisData(symbol, decl, semanticModel, analysisScope, cancellationToken);
+            using var declarationAnalysisData = AnalyzerDriver<TLanguageKindEnum>.ComputeDeclarationAnalysisData(symbol, decl, semanticModel, analysisScope, cancellationToken);
```

Comment:
I didn't understand why the `AnalyzerDriver` qualifier is now necessary.

Reply:
fixed!